npm/puppeteer/README.md

```diff
@@ -0,0 +1,368 @@
+# @cypress/puppeteer
+
+Utilize [Puppeteer's browser API](https://pptr.dev/api) within Cypress with a single command.
+
+# Table of Contents
+
+- [Installation](#installation)
+- [Compatibility](#compatibility)
+- [Usage](#usage)
+- [API](#api)
+- [Examples](#examples)
+- [Contributing](#contributing)
+- [Changelog](./CHANGELOG.md)
+
+# Installation
+
+## npm
+
+```sh
+npm install --save-dev @cypress/puppeteer
+```
+
+## yarn
+
+```sh
+yarn add @cypress/puppeteer
+```
+
+## With TypeScript
+
+Add the following in `tsconfig.json`:
+
+```json
+{
+  "compilerOptions": {
+    "types": ["cypress", "@cypress/puppeteer"]
+  }
+}
+```
+
+## Compatibility
+
+`@cypress/puppeteer` requires Cypress version 13.6.0 or greater.
```

Comment:
Do we want to add a `peerDependency` in the `package.json` for this requirement?

Reply:
Good idea. Addressed [here](https://github.com/cypress-io/cypress/pull/28370/commits/beb85942ff50c82e10405d142cc6b0bc820580c6#diff-dbb2ec8f0a23ac83b2225ec8d8459d75e8490d25b7f146ea51de157bbd9815f6R35-R37)